From 210c9e5dd1e3d0e37a16225a63840d71b473684c Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Mon, 15 Jun 2015 12:05:11 +0200
Subject: [PATCH] man: document that SIGCONT always follows SIGTERM

As requested in #199.

(cherry picked from commit e8c53936316288ea3b33b5997b175862f0efef92)
Resolves: #1601794
---
 man/systemd.kill.xml | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/man/systemd.kill.xml b/man/systemd.kill.xml
index e57f0e724..1292f4f51 100644
--- a/man/systemd.kill.xml
+++ b/man/systemd.kill.xml
@@ -136,7 +136,13 @@
         by <constant>SIGKILL</constant> (see above and below). For a
         list of valid signals, see
         <citerefentry project='man-pages'><refentrytitle>signal</refentrytitle><manvolnum>7</manvolnum></citerefentry>.
-        Defaults to <constant>SIGTERM</constant>. </para></listitem>
+        Defaults to <constant>SIGTERM</constant>. </para>
+
+        <para>Note that right after sending the signal specified in
+        this setting systemd will always send
+        <constant>SIGCONT</constant>, to ensure that even suspended
+        tasks can be terminated cleanly.</para>
+        </listitem>
       </varlistentry>
 
       <varlistentry>
